README.md

```diff
@@ -201,5 +201,90 @@ to check that it hasn't crashed.
 
 Congratulations, you have just run your first GRChombo simulation!
 
+### Visualisation
+
+In this section, we will outline how to get set up with visualisation using
+Paraview on Fawcett. It is a bit fiddly, so we will go over the bare minimum
+to get up and running. For more detailed information on what Paraview can do,
+have a look at the
+[tutorial](https://www.paraview.org/Wiki/images/b/bc/ParaViewTutorial56.pdf)
+(or ask Amelia!). Adapted from instructions provided by Kacper Kornet.
+
+First, we need to set up ssh access to fawcett. Information on how to do this
+is given [here](https://www.maths.cam.ac.uk/computing/fawcett-ssh-access). We
+have sent this to you in advance, so you should have all of the steps set up
+prior to the tutorial.
+
+Second, we need you to download Paraview v5.6.0 onto your local machine i.e. the
+laptop/desktop you are using to access fawcett. This can be done
+[here](https://www.paraview.org/download/). Again, we will have asked you to
+do this in advance.
+
+Before we tunnel to fawcett, we need to add a server on the Paraview
+client. To do this, open Paraview, and click 'connect' in the top left corner,
+then `add server'. Change the name to whatever you like e.g. 'localhost'. All
+other settings should be the default
+
+Server type: Client/Server
+Host: localhost
+Port: 11111
+
+Click Configure. On the next screen, set 
+
+Startup Type: Manual
+
+Accept by clicking Save and Close. Once you have set this up, you won't need
+to do it again.
+
+Now we need to start the Paraview server on fawcett. Ssh into fawcett - as
+outlined above, this should already be configured with ssh forwarding, and so
+shouldn't require explicit login via another machine. Load Paraview using
+
+```bash
+module load paraview/5.6.0/upstream
+```
+
+Now run the command
+```bash
+pvserver
+```
+
+It should print something like:
+
+```bash
+Waiting for client...
+Connection URL: cs://fawcett:11111
+Accepting connection(s): fawcett:11111
+```
+
+Note that the number 11111 can be different if someone else
+is already running pvserver. From here on, we will call it `cosmos2_port`.
+
+(For large datasets, it is a good idea to submit a job to the queue and run
+pvserver with a modified environment
+
+```bash
+OSPRAY_THREADS=8 KNOB_MAX_WORKER_THREADS=8 pvserver
+```
+
+where 8 is the number of cores reserved for the job. However for the purposes
+of this tutorial we should be fine without.)
+
+We next need to set up the tunnel to fawcett. On your local machine, run
+
+```bash
+ssh -NL 11111:localhost:cosmos_port username@fawcett.maths.cam.ac.uk
+```
+
+It should ask for authentication, then look as though it has 'hung up'.
```

Comment:
With ssh keys set up, I was not prompted for authentication.
```suggestion
It may ask for authentication and then look as though it has "hanged" (i.e. no prompt).
```

Reply:
"Hanged" has a different meaning!

```suggestion
It should ask for authentication, then look as though it has 'hung'.
```